numpy/random/mtrand.pyx

```diff
@@ -3639,6 +3640,17 @@ cdef class RandomState:
         tol : float, optional
             Tolerance when checking the singular values in covariance matrix.
             cov is cast to double before the check.
+        mode : { 'default', 'legacy', 'fast', 'factor'}, optional
+            The method used for sampling. The default option uses eigen
```

Comment:
`s/eigen/eigh` - you may as well be specific about the exact implementation which is used